napari_btrack/track.py

```diff
@@ -31,23 +151,276 @@ def run_tracker(objects, config_file_path):
         return data, properties, graph
 
 
-def track() -> FunctionGui:
-    @magicgui(
-        call_button=True,
-        persist=True,
-        config_file_path=dict(value=Path.home()),
-        reset_button=dict(widget_type="PushButton", text="Reset defaults"),
+def get_save_path():
```

Comment:
It feels like it would be much better to have the application as a parameter to the function (similarly for `get_load_path()`) instead of relying on `use_app()` to get the right application.

Reply:
I haven't found a good way around this. Leaving for now at least.